### PR TITLE
Makes several adjustments to Ash Walkers

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Removes the 'NOBREATH' trait from Ash Walkers and instead gives them unique lungs that are adapted to breathing the atmosphere on Lavaland. Said lungs also find the concentration of O2 in normal environments too high and will start choking when breathing it in. Adds the TRAIT_RESISTLOWPRESSURE, mainly to remove the 'low pressure' screen warning. Ashies are very unlikely to find themselves in a situation where this trait will actually be of use.
Also makes the Gladiator uniform display digitigrade legs and replaces the atmosphere around the Ash Walker tendril with Lavaland's atmosphere.
Also headcoder approval : 
image

## Why It's Good For The Game
For some reason lizards native to lavaland don't need to breathe it all instead of being able to breathe Lavaland's atmosphere normally. Also, them having an adverse reaction to the station's air mix enforces the policy that ashies should not go on station.
And if they're brought there forcefully, their captors can just give them a lung transplant. The constant 'low pressure' alert was tad annoying, and odd to have for a species that adapted to Lavaland.
The gladiator uniform not displaying digitigrade legs was just stupid, and the area around the tendril having normal station air was odd.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Breathing normally on Lavaland
image

Choking on station
image

</details>

## Changelog
:cl:
add: Added a new lung subtype - Ash Walker lungs, that are adapted to low pressure, low O2 and high CO2 concentrations
tweak: Made the Gladiator Uniform display digitigrade legs
tweak: Replaced the station air around the Ash Walker tendril with lavaland air
balance: Made Ash Walkers have Lavaland adapted lungs rather than not needing to breathe at all
balance: Ash Walkers are now low pressure immune
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
> `TRAIT_RESISTLOWPRESSURE`

Is this not tied to their lungs and the pressure requirements of that? Or does it specifically relate to pressure-related brute damage?
A trait just to hide a warning seems incorrect - the warning should only appear if the pressure is too low for their lungs or causes brute damage.

**Comment by itsmeow** [comment]
I see that the trait already exists. So I suppose my criticism is invalid, but are you sure there's not another way to prevent the warning?

**Comment by CydiaLamiales** [comment]
> but are you sure there's not another way to prevent the warning?

Not without some majorly snowflake checks
image
image

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -100,6 +100,7 @@
 	id = SPECIES_ASHWALKER
 	examine_limb_id = SPECIES_LIZARD
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS, NO_UNDERWEAR)
-	inherent_traits = list(TRAIT_NOGUNS,TRAIT_NOBREATH)
+	inherent_traits = list(TRAIT_NOGUNS,TRAIT_RESISTLOWPRESSURE)
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: honestly I think I'd rather them just have the low pressure warning than fully resist 0 pressure, if theres no way to add partial low pressure resistance (its part of /mob/living but I guess its not accessible via species)

Comment (CydiaLamiales) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: As far asI know there really isn't a way to do that unless we introduce a new trait. Guess I'll remove it then.

Comment (CydiaLamiales) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]:
```suggestion
	inherent_traits = list(TRAIT_NOGUNS)
```

Comment (CydiaLamiales) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: There, removed.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Hopefully there isnt BZ in the atmosphere that messes them up or something

**Comment by CydiaLamiales** [comment]
Fairly certain Lavaland doesn't get random gasses anymore.

**Pull request merged**